packages/test/end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -33,6 +35,26 @@ class TestSharedDataObject1 extends DataObject {
     public get _context() {
         return this.context;
     }
+
+    public async request(request: IRequest): Promise<IResponse> {
+        const url = request.url;
+        const parsed = parse(url, true);
+        const requestParser = RequestParser.create({ url: request.url });
+        const itemId = requestParser.pathParts[0];
+        if (itemId === "bigBlobs") {
+            const value = this.root.get<string>(requestParser.pathParts.join("/"));
+            if (value === undefined) {
+                return { mimeType: "text/plain", status: 404, value: `request ${url} not found` };
+            }
+            return { mimeType: "fluid/object", status: 200, value };
+        // eslint-disable-next-line no-null/no-null
+        } else if (parsed.search !== null) {
+            return { mimeType: "fluid/object", status: 200, value : `${parsed.search}` };
```

Comment:
Please add a comment explaining why you are returning this instead of the data object itself.